pyproject.toml

```diff
@@ -101,6 +101,8 @@ include = [
         # Don't complain if tests don't hit assertions
         "raise AssertionError",
         "raise NotImplementedError",
+        # Don't complain about connection errors
+        "raise requests.ConnectionError",
         # Don't complain about script hooks
```

Comment:
Is this to avoid errors during testing if the TESS servers are not available?

Reply:
Yes, and I wasn't sure how to test for that case...